Blog/views.py

```diff
@@ -66,32 +66,33 @@ def post_detail_view(request, pk):
 
 
 # Inserts a lot of users, posts and comments to the db
-# def insert_to_db(request):
-#     blogpost_content = 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.'
-#     comment_content = 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyamLorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam'
-#
-#     for user_count in range(100):
-#         user = User()
-#         user.username = "test-username-{}".format(str(uuid4()))
-#         user.first_name = "test-first-{}".format(str(uuid4()))
-#         user.last_name = "test-last-{}".format(str(uuid4()))
-#         user.save()
-#         user.refresh_from_db()
-#         for blogpost_user_count in range(random.randint(1, 30)):
-#             post = Post()
-#             post.created = datetime.datetime.now()
-#             post.title = "title-{}".format(str(uuid4()))
-#             post.content = blogpost_content
-#             post.user = user
-#             post.save()
-#             post.refresh_from_db()
-#             for blogpost_comment_count in range(random.randint(1, 20)):
-#                 comment = Comment()
-#                 comment.content = comment_content
-#                 comment.user = user
-#                 comment.created = datetime.datetime.now()
-#                 comment.post = post
-#                 comment.save()
-#     return index(request)
-
-
+def insert_to_db(request):
+    if request.user.is_superuser is False:
```

Comment:
I tend to make the assertion the other way round: Third party code changes all the time. What if `request.user.is_superuser` returns `0` instead of `False` in a future release? Then the assertion would be true and the data gets inserted.

So it's safer to check for `if not request.user.is_superuser is True` or in short `if not request.user.is_superuser`.